<launch>
<!-- Launch file for Livox AVIA LiDAR -->

  <rosparam command="load" file="$(find std_detector)/config/config_ntu.yaml" />

  <node pkg="std_detector" type="generate_database" name="generate_database" output="screen"/>

  <param name="generate_descriptor_database" type="bool" value = "true"/>
  
  <param name="generated_lidar_path" type="string" value = "/home/dat/data/std_reloc/priormap/NTU/pointclouds/"/>
  <param name="generated_pose_path"  type="string" value = "/home/dat/data/std_reloc/priormap/NTU/kfpose_fixed.pcd"/>
  <param name="descriptor_file_path" type="string" value = "/home/dat/data/std_reloc/priormap/saved_descriptors/"/>
  <param name="kf_merged"            type="int"    value = "2"/>

  <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find std_detector)/rviz_cfg/demo.rviz" />

</launch>